### PR TITLE
Laravel 13.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 13.x.

**Before merging**, you need to:

- Checkout the `l13-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/laravel-health-check.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "ans-group/laravel-health-check": "dev-l13-compatibility",
    }
}
```

Finally, run: `composer update`

File: composer.json [hunk 1]
```diff
@@ -57,27 +57,27 @@
...
-        "illuminate/redis": "^10.0|^11.0|^12.0",
+        "illuminate/database": "^10.0|^11.0|^12.0|^13.0",
+        "illuminate/log": "^10.0|^11.0|^12.0|^13.0",
+        "illuminate/redis": "^10.0|^11.0|^12.0|^13.0",
         "larastan/larastan": "^3.1",
         "league/flysystem": "^3.29",
         "league/flysystem-ftp": "^3.29",
         "mockery/mockery": "^1.0",
-        "orchestra/testbench": "^8.0|^9.0|^10.0",
+        "orchestra/testbench": "^8.0|^9.0|^10.0|^11.0",
```
[the remark this answers is not in the document]
Comment (phily245) [reply composer.json:74]: @copilot can you take a look? We need to test from the minimum versions of PHP for each Laravel version all the way upto 8.5

**Comment by agpopov** [comment]
up

**Comment by phily245** [comment]
This needs the changes suggested by Copilot doing, however, I don't have permissions to write to this branched fork. I'll raise a new PR with these changes

**Comment by phily245** [comment]
A new PR has been raised at https://github.com/ans-group/laravel-health-check/pull/116

**Comment by phily245** [comment]
Closing in favour of #116
